lib/ui/shared/preferences/preferenceswidgets/notification/preferences_notification_viewmodel.dart

```diff
@@ -1,8 +1,16 @@
+import 'dart:convert';
 import 'package:stacked/stacked.dart';
+import 'package:zc_desktop_flutter/app/app.locator.dart';
 import 'package:zc_desktop_flutter/core/enums/flash_windows.dart';
 import 'package:zc_desktop_flutter/core/enums/pref_message.dart';
+import 'package:zc_desktop_flutter/models/preferences_model/notification/notification.dart';
+import 'package:zc_desktop_flutter/services/local_storage/local_storage_service.dart';
 
 class NotificationViewModel extends BaseViewModel {
+  final _localStorage = locator<LocalStorageService>();
+  final _notificationStorageKey = 'notificationSetting';
```

Comment:
Why are you defining the key here in the viewmodel, move it to the sharedPrefrences set up to keep consistency

Reply:
done